cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -101,30 +101,22 @@ func (cm *certManagerClient) setManifestHash() error {
 }
 
 func (cm *certManagerClient) setManifestVersion() error {
-	// Gets the cert-manager objects from the embedded assets.
-	objs, err := cm.getManifestObjs()
+	// Gets the cert-manager version from the image version in the raw yaml
+	yaml, err := manifests.Asset(embeddedCertManagerManifestPath)
 	if err != nil {
 		return err
 	}
-	found := false
 
-	for i := range objs {
-		o := objs[i]
-		if o.GetKind() == "CustomResourceDefinition" {
-			labels := o.GetLabels()
-			version, ok := labels[certmanagerVersionLabel]
```

Comment:
We should remove the const `certmanagerVersionLabel = "helm.sh/chart"` from line 54 above.